CTP/ha_repl/src/com/navercorp/cubridqa/ha_repl/Test.java

```diff
@@ -686,7 +686,7 @@ private String executeSQL(String sql) {
 	private Connection getDBConnection() throws SQLException {
 		if (connection == null || connection.isClosed()) {
 			String host = hostManager.getInstanceProperty("master." + ConfigParameterConstants.TEST_INSTANCE_HOST_SUFFIX);
-			String port = hostManager.getInstanceProperty("broker.BROKER_PORT");
+			String port = hostManager.getAvailablePort("master");
 			String url = "jdbc:cubrid:" + host + ":" + port + ":" + hostManager.getTestDb() + ":::";
 			connection = DriverManager.getConnection(url, "dba", "");
```

Comment:
It's better to make test case fail once fail to get a connection.
Current codes will miss such error.